pkg/validate/validate.go

```diff
@@ -176,3 +177,11 @@ func MAC(addr string) error {
 	_, err := net.ParseMAC(addr)
 	return err
 }
+
+// IPNotInCIDR validates that a given IP is not contained within a CIDR subnet
+func IPNotInCIDR(ip string, cidr ipnet.IPNet) error {
```

Comment:
why the `Not`, most of other functions in the package seem to validate positive conditions?

Reply:
Since baremetal does not want the VIPs in the subnets/ranges being tested against, it made sense to me to validate the negative. Otherwise, the caller would have to negate the test and come up with its own error message. Just seems simpler to me... But I'm fine with changing it.

Reply:
I stand corrected. Upon further investigation, the VIPs *should* be in MachineCIDR, but should *not* be in the others. I will rework accordingly.